public/content/roadmap/dencun/index.md

```diff
@@ -0,0 +1,151 @@
+
+## Does this upgrade affect all Ethereum consensus and validator clients? {#client-impact}
+
+Yes. Proto-Danksharding (EIP-4844) involves changes to both the execution clients _and_ consensus clients. All production Ethereum clients have released updates to support the upgrade. Node operators must upgrade to a supported version to stay in sync with the chain after the upgrade. <!-- TIME-SENSITIVE --> [See details on supported client releases](https://blog.ethereum.org/2024/02/27/dencun-mainnet-announcement#client-releases)
+
+_Validator_ software is handled by the consensus clients, which have all been updated to accommodate the upgrade.
+
+## How does Cancun-Deneb (Dencun) affect Goerli or other Ethereum testnets? {#testnet-impact}
+
+- Devnets, Goerli, Sepolia and Holesky have all undergone the Dencun upgrade and have Proto-Danksharding fully functioning
+- Rollup developers can use these networks for EIP-4844 testing
+- Most users will be completely unaffected by this change to each testnet
+
+## What is the economic impact for the included Ethereum Improvement Proposals (EIPs)? {#economic-impact}
+
+### Cancun (Execution) {#economic-impact-cancun}
+
+[EIP-4844](https://eips.ethereum.org/EIPS/eip-4844) - *Shard blob transactions (Proto-Danksharding)*
+
+- Introduction and utilization of blob-space reduces L1 gas demand in the near-term, making all transactions less expensive on average ([see L1 fee impact below](#l1-fee-impact))
+- This upgrade primarily affects L2 rollup fees, making them significantly cheaper, but also helps offload some demand for L1 block space
+- Lower L1 gas costs means the amount of ETH burned from transaction fees is also reduced, making ETH less deflationary
+- Lower gas costs make the network more accessible to new users and new use cases, which may reflexively increase demand for block space over time
+
+[EIP-7516](https://eips.ethereum.org/EIPS/eip-7516) - *Blob base fee opcode*
+
+- Introduces a new fee market, similar to the EIP-1559 mechanism, dedicated to pricing blob space
+- This helps offload demand for traditional L1 block space, reducing fees and reducing the amount of ETH burned on average
+
+### Deneb (Consensus) {#economic-impact-deneb}
+
+[EIP-4844](https://eips.ethereum.org/EIPS/eip-4844) - [See above](#economic-impact-cancun)
+
+[EIP-7044](https://eips.ethereum.org/EIPS/eip-7044) - *Perpetually valid signed voluntary exits*
+
+- This reduces the trust assumptions for users of pooled staking or staking-as-a-service platforms by providing a self-sovereign means of exiting validator funds without any permission from a third-party node operator
```

Comment:
```suggestion
- This reduces the trust assumptions for users of pooled staking or staking-as-a-service platforms by providing a perpetual trustless means of exiting validator funds without any permission from a third-party node operator
```

You can already do "self sovereign" exits, the difference with 7044 is those exit messages no longer expire.